Tools/ros2/ardupilot_msgs/msg/Rally.msg

```diff
@@ -0,0 +1,13 @@
+geographic_msgs/GeoPoint point
+float32 break_altitude # when autolanding, break out of loiter at this alt (meters)
+float32 land_dir # when the time comes to auto-land, try to land in this direction (degrees)
```

Comment:
What coordinate system? I assume 0 is south to north, and 45 is south west to north east (compass bearing coordinates)

Reply:
I think I need to add more documentation from the library to the comments there.

Reply:
That could be good. AP libraries can be a bit light on docs.

Reply:
I should probably unpack the flags in the message, to make it easier to set from ROS.

Reply:
Might be nice for ROS, but I would double check the flash impact before comitting to it. 

An alternative is to supply some C++/python functions in ardupilot_msgs to unpack/pack the data.

Reply:
I have unpacked the flags to make it easier from ROS to interact with those settings.